cmd/apps/skychat/chat.go

```diff
@@ -34,40 +35,28 @@ var addr = flag.String("addr", ":8001", "address to bind")
 var r = netutil.NewRetrier(50*time.Millisecond, 5, 2)
 
 var (
-	chatApp   *app.Client
-	clientCh  chan string
-	chatConns map[cipher.PubKey]net.Conn
-	connsMu   sync.Mutex
-	log       *logging.MasterLogger
+	log      = logrus.New()
```

Comment:
@nkryuchkov As we are displaying logs of apps within visor, I didn't want to end up with something like this:
```
[2020-05-11T03:46:57+12:00] INFO [proc_manager]: [2020-05-11T03:46:57+12:00] DEBUG []: This is an example log.
```
Also, the standard `logrus.TextFormatter` format is easier to parse than in `logging`. This will be helpful as we need to display logs in `hypervisor`.

Reply:
@evanlinjin Thank you for the explanation.